.github/workflows/website.yml

```diff
@@ -33,25 +33,39 @@ jobs:
         run: |
           pip install -r requirements.txt
 
-      # Action to download artifacts from a different workflow (analysis.yml)
+      # Action Figures artifact
       - name: 'Download artifact'
         if: ${{ github.event.workflow_run.conclusion == 'success' }}
         uses: ./.github/actions/download-artifact
         with:
           name: 'Figures'
+      # Action analysis data artifact
+      - name: 'Download analysis data'
+        if: ${{ github.event.workflow_run.conclusion == 'success' }}
+        uses: ./.github/actions/download-artifact
+        with:
+          name: 'Data'
 
-      - name: Build html
+      - name: move data to the website folder
+        run: |
+          mv test_output.csv website/
+
+      - name: Build documentation
         run: |
           mkdir docs/_static
           mv *.pdf docs/_static/
           sphinx-apidoc -o docs src
           cd docs/
           make html
 
+      - name: move data to the website folder
+        run: |
+          mv docs/_build/html website/documentation
```

Comment:
Is this a space in the folder name? Probably should avoid that for safety.

Reply:
done